docs/DeveloperGuide.md

```diff
@@ -86,9 +86,9 @@ The `UI` component,
 1. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
 1. In addition, the `CommandResult` object can also instruct the `Ui` to perform certain actions, such as displaying help to the user.
 
-Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete 1")` API call.
+Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete A1234567X")` API call.
```

Comment:
Does this API call take a string as the argument, or the model?

Reply:
Resolved! I changed it to "...for the `delete A1234567X` Command API call"